src/azure-cli/azure/cli/command_modules/vm/generated/_help.py

```diff
@@ -0,0 +1,65 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['vm ssh-public-key'] = """
+    type: group
+    short-summary: Manage ssh public key with vm
+"""
+
+helps['vm ssh-public-key list'] = """
```

Comment:
seems it's better Aladdin examples could be moved to /generated/_help.py also

Reply:
Agree